tests/test_database_redis.py

```diff
@@ -100,6 +100,15 @@ class TestRedisDatabaseAsync(asynctest.TestCase):
 
         result = await db.put("string", dict(key="value"))
 
+    async def test_delete(self):
+        db = RedisDatabase({})
+        db.client = MockRedisClient()
+        db.client.execute = amock.CoroutineMock(return_value=None)
```

Comment:
I was having some CI issues with this pattern yesterday. Could you change it to the following, just for peace of mind.

```suggestion
        db.client.execute = amock.CoroutineMock()
        db.client.execute.return_value = None
```

Reply:
Made the changes as mentioned.